selfdrive/car/tests/test_docs.py

```diff
@@ -22,14 +21,19 @@ def test_generator(self):
     generated_cars_md = generate_cars_md(self.all_cars, CARS_MD_TEMPLATE)
     with open(CARS_MD_OUT) as f:
       current_cars_md = f.read()
-
     assert generated_cars_md == current_cars_md, "Run selfdrive/car/docs.py to update the compatibility documentation"
 
   def test_docs_diff(self):
-    dump_path = os.path.join(BASEDIR, "selfdrive", "car", "tests", "cars_dump")
-    dump_car_docs(dump_path)
+    dump_path = os.path.join(BASEDIR, "selfdrive", "car", "tests", "cars_dump", "CARS.md")
+    file_path = os.path.join(dump_path, "CARS.md")
+    def download_file(url, filename):
+      import requests
+      response = requests.get(url)
+      with open(filename, 'wb') as file:
+        file.write(response.content)
+    download_file("https://raw.githubusercontent.com/commaai/openpilot/master/docs/CARS.md", file_path)
```

Comment:
this test simply checks that the file is up to date with any changes on the branch, you shouldn't need to change anything